### ISSUE TITLE
Servo jittering after disarm

### ISSUE THREAD

**Issue by roman-dvorak** [PX4/Firmware#14900]
**Describe the bug**
After disarm the servo actuator connected to MAIN1 output starts jittering. The intensity of this jittering varies. This state occurs only sometimes. The tested setup was CUAV Nano FMUv5 without IO co-processor. I have observed this behaviour with our Autogyro airframe. But it was repeated with a standard airplane airframe. It doesn't matter what the mixer is.

Firmware was master from 6.5.

**To Reproduce**
Steps to reproduce the behaviour:
1. Connected power on PPM ports and connect some servo to MAIN1 port
2. Switch autopilot on
3. Arm and disarm autopilot
4. See error - servo start jitter

**Video**
This video was recorded after standard disarm through RC
https://youtu.be/r_J6SnLum-c

This signal is only on 1st MAIN output. It looks like the same 'random' signal. It has been verified that this is not a power interference. A variable signal is really output from this port. It looks like the output is set from some memory location, which is overwritten by other values.

We were also surprised that the servos are not 'active' (no pwm output) after pressing the safety button. Only after the ARM. Is this behaviour normal with an autopilot without an IO processor?

**Comment by dagar** [comment]
Do you have the corresponding log?

**Comment by roman-dvorak** [comment]
I have a log that ends with disarming before starting the output jittering.. I haven't log directly with that jittering (I'll try to make one).

Sometimes it starts immediately after disarm. Sometime it takes some time (for example 10 minutes) and intensity is rising. 

https://review.px4.io/plot_app?log=60ca6141-07e0-4d88-8439-c661b4b81807

**Comment by roman-dvorak** [comment]
I have created a new log and video:

https://review.px4.io/plot_app?log=0818c8ca-c9c5-4e3d-a841-8c612e26adef
https://youtu.be/vTO9dVAxjyM

Now I noticed strong noise of the magnetometer.

**Comment by dagar** [comment]
I just wanted to confirm that PX4 wasn't sending anything. 

![Screenshot from 2020-05-14 10-45-57](https://user-images.githubusercontent.com/84712/81948822-27b30000-95d0-11ea-9dba-41af63aad942.png)

Does it stop if you unplug the telemetry radio?

**Comment by roman-dvorak** [comment]
@julianoes 
Hi, today I got to the oscilloscope and I wanted to measure what the jittering signal look like.

So I connected oscilloscope probe to signal pin of MAIN1 output. Against ground from PPM header. The telemetry modem was disconnected during the measurement.  I armed autopilot and oscilloscope show nice PWM output with 3.3V logic. 


![NewFile1](https://user-images.githubusercontent.com/5196729/82740626-f910f400-9d4a-11ea-94fb-ac633eb0821a.png)

Then I disarmed autopilot and oscilloscope shows a problematic signal that was causing servo jittering:
![NewFile2](https://user-images.githubusercontent.com/5196729/82740647-29f12900-9d4b-11ea-9211-07577b2707d4.png)

Sometimes it stays for few seconds in logic 1 and sometimes zero-oriented peaks were significantly more frequent (without screenshot)

![NewFile0](https://user-images.githubusercontent.com/5196729/82740704-8c4a2980-9d4b-11ea-9295-1cca8a47cba0.png)

Any other pin was in login 0. 
![NewFile3](https://user-images.githubusercontent.com/5196729/82740714-a6840780-9d4b-11ea-9789-bc192887ba65.png)



That's why I think it's some bug.

**Comment by roman-dvorak** [comment]
@dagar please, can you advise if it or it is not a software bug. Based on measurements from the last comment? I think there's something uninitialized. Or something is being written to the wrong place in memory.

**Comment by dagar** [comment]
This is a CUAV v5 nano (no px4io) and it's only ever the first pin?

**Comment by roman-dvorak** [comment]
Yes, it is only on CUAV v5 nano without the IO coprocessor. And it is only on the 1st pin labeled as MAIN1. 

I have trayed upload the same firmware build to CUAV v5+ (with px4io) and this issue was not observed.

**Comment by Antiheavy** [comment]
@roman-dvorak do you have a 900 MHz radio (915 MHz) plugged into your system?  does moving the radio or antenna change things?  just curious....

**Comment by roman-dvorak** [comment]
During the first test yes. But 433MHz radio from holybro. The disconnecting modem doesn't affect this. It was tested after some of the previous comment.

Nothing was connected to autopilot when measuring with an oscilloscope. I have trayed more power sources. USB from two computers, powering through power module and LiPo. Behavior was the same.

It is interesting that the 1st pin is in log1. And "peaks" are grounded (to log 0). Other outputs are still in log 0.

**Comment by AlexandreBorowczyk** [comment]
@roman-dvorak 
Did you manage to find a solution? I believe we have the same issue. I have check with a oscilloscope but the servo motion is the same

**Comment by roman-dvorak** [comment]
Hi, no I haven't fixed it and I don't even know if it still occurs with today's FW.
I solved it by not using the first PPM output. Others do not suffer from this problem. Fortunately, we had some outputs free.

**Comment by AllenHuangDongyue** [comment]
The same occured on me. Is there any solutions?
